### ISSUE TITLE
[Bug][iOS]: Started playback when I wanted to accept a call

### ISSUE THREAD

**Issue by elmoritz** [gdavis/Mimir-Feedback#105]
### Are you sure this is a new issue?

- [X] I have confirmed this is a new issue

### iOS Version

15.4.1

### Device Model

iPhone 13 Pro Max

### App Version

1.0.0 (84)

### What happened?

I was listening to a podcast, when a call came in.
I pressed the accept-call button on my AirPods Pro.
The Call got accepted AND the podcast continued playing.
It was a strange situation, since I was in the Car and had to open the app to pause the podcast.

### Bug prevalence

only once, when receiving a Call

### How does it happen?

I will describe the situation without the car but just with the AirPods, since I don't believe it will make a difference.

1. start a podcast in Mimir
2. wait for a call
3. answer the call by pressing the AirPods once
4. Observe

I have not had this Issue with any other player.

### Anything else?

I hope this gets fixed before Ragnarok

**Comment by gdavis** [comment]
Hmm interesting. I usually have noticed the player properly pausing when a call comes in, but looks like something was missed. Will look into.

**Comment by gdavis** [comment]
Also being tracked here: gdavis/Mimir#778

**Comment by gdavis** [comment]
Tested this scenario a few times today and was unable to have the same situation occur. This could be due to the changes I've made recently to the audio session, so we'll need to circle back on this one and retest with the latest version of Mimir.

**Comment by gdavis** [comment]
This latest update had some changes around how the player handles interruptions. I'm going to go ahead and close this issue until it happens again.
